src/discordcr/mappings/rest.cr

```diff
@@ -15,5 +15,27 @@ module Discord
         pruned: UInt32
       )
     end
+
+    # A response to the Modify Webhook REST API call.
+    class ModifyWebhookPayload
+      JSON.mapping(name: String?, avatar: String?, channel_id: UInt64?)
```

Comment:
Please make the formatting here consistent with all the other `JSON.mapping` calls (newlines)